charts/govuk-rails-app/values.yaml

```diff
@@ -29,7 +29,7 @@ workerReplicaCount: 1
 dbMigrationEnabled: false
 
 appImage:
-  repository: ""  # Dummy value, overridden in ArgoCD config.
+  repository: "172025368201.dkr.ecr.eu-west-1.amazonaws.com/dummy-value"  # Dummy value, overridden in ArgoCD config.
```

Comment:
Does this defeat the `required` in the template?

Reply:
Yep good point. I'll remove the `required` constraint and set this to `""`. Hopefully we have a constraint in the schemas.